src/cmd/support_archive/logs.go

```diff
@@ -0,0 +1,123 @@
+package support_archive
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
```

Comment:
`v1` is ambiguous, maybe `clientgov1`

Reply:
clientgo is a bit broad 
could be `corev1types` (which is a mouthful)